src/css/organisms/homepage-header.scss

```diff
@@ -1,24 +1,29 @@
 .homepage-header {
-  background: url('/static/img/homepage-header-pattern.svg') $color-gray-20;
+  background: url('/static/img/homepage-header-pattern.svg') $color-black;
   background-position: calc(50% + 400px);
```

Comment:
Just out of curiosity, why not roll these lines into the `background` above? Compatibility for `calc()`?

Reply:
Good shout! Updated 👌